showcase/src/app/shared/shared.module.ts

```diff
@@ -44,7 +45,8 @@ import { FileService } from "./services";
 		MatTooltipModule,
 		MatSnackBarModule,
 		StarkPrettyPrintModule,
-		TranslateModule
+		TranslateModule.forChild(),
```

Comment:
If we remove the `translate.instant` call in the ExampleViewer, then this can be removed too